tools/ci_build/github/windows/requirements_eager.txt

```diff
@@ -0,0 +1,9 @@
+setuptools
```

Comment:
Please rename the file to eager/requirements.txt

Reply:
Done.